docs/schemas/2024-04/donor-details.json

```diff
@@ -212,16 +212,7 @@
         }
       ],
       "type": "object",
-      "required": ["dateOfBirth", "email", "status", "channel"],
-      "if": {
-        "required": ["channel"],
-        "properties": {
-          "channel": { "const": "online" }
-        }
-      },
-      "then": {
-        "required": ["email"]
-      },
+      "required": ["dateOfBirth", "email", "status"],
```

Comment:
but as Greg says, you may want to remove email (and the existing validation in create) so you can do the consumer side